employee-rostering-frontend/src/ui/components/calendar/EventWrapper.tsx

```diff
@@ -31,17 +31,14 @@ export default function EventWrapper(props: React.PropsWithChildren<{
   popoverBody: React.ReactNode;
   style?: React.CSSProperties;
 }>): JSX.Element {
-  let className = (props.className) ? props.className : 'rbc-event';
+  let className = (props.className) ? props.className : '';
```

Comment:
`props.className` can't be null (it's `string`) so the check is redundant. Can be simplified to
```ts
let { className } = props;
```